pybleau/app/plotting/axis_style.py

```diff
@@ -49,7 +49,7 @@ class AxisStyle(HasStrictTraits):
 
     #: Is there a finite number of labels?
     # (If so, allow forcing the appearance of all of them and label rotation)
-    _finite_labels = Bool
+    support_text_labels = Bool
```

Comment:
Does the docstring for this need to change, now?

Reply:
Yes!